weaviate-services/agents/transformation-agent-get-started.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Wdyt about adding comments:</p><ul><li>Saying that this will create properties in the <code>ArxivPapers</code> as needed during import, using Weaviate's <a href="https://weaviate.io/developers/weaviate/config-refs/schema#auto-schema" target="_blank">Auto-schema functionality</a>.</li></ul><p> </p><p>I think the <code>Property</code> &amp; <code>DataType</code> classes are unused here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/weaviate/recipes/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>